### ISSUE TITLE
Scanty Documentation

### ISSUE THREAD

**Issue by christopherokonkwo** [AndreasElia/laravel-firefly#40]
The documentation doesn't say much about how it works, whether it bootstraps its own frontend or is frontend agnostic. It would make sense to have a mental picture of the package before installing and what better way to archive that if not the Docs?

**Comment by AndreasElia** [comment]
A very good point, and it's one of the reasons I have really advertised it too much just yet. I'll make updating the documentation one of my tasks for tomorrow 😄

**Comment by AndreasElia** [comment]
Good afternoon @christopherokonkwo, hope you're doing well.

I've updated the README with a comment regarding the templates/publishables. I have also included some screenshots below after I pushed a couple of frontend spacing fixes I noticed whilst preparing them.

![image](https://user-images.githubusercontent.com/5033092/114312242-eed62080-9ae9-11eb-8d02-e80ebbdeb44d.png)
![image](https://user-images.githubusercontent.com/5033092/114312264-fc8ba600-9ae9-11eb-8e2d-2f49568c30ac.png)
![image](https://user-images.githubusercontent.com/5033092/114312276-08776800-9aea-11eb-9c3c-e75ee4936a1b.png)

Everything can be overwritten, and I'd probably advise people do so in order to tie things into their own templates better, but if you'd like to see any changes or features - feel free to post an issue for them 😄
